<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualDialog</class>
 <widget class="QDialog" name="ManualDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Suction</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>254</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="BTN_SUCTION_RESET">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BNT_SUCTION">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>suction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_SUCTION_RELEASE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>release</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Joint/TCP move</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;JOINT MODE [deg]&lt;/p&gt;&lt;p&gt;[joint1,2,3,4,5,6]&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;TCP MODE [mm,deg]&lt;/p&gt;&lt;p&gt;[x,y,z,roll,pitch,yaw]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE1">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE2">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE3">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE4">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE5">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LE_COORDINATE6">
         <property name="text">
          <string>0.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>151</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;speed [0~1]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LE_SPEED">
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;acc [0~1]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="LE_ACC">
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="BTN_COL_RESUME">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Collision
Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_MOTION_RESUME">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>160</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Motion
Resume</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_TCP">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="BTN_MOTION_HALT">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Halt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_MOTION_MOVE">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Move</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RB_JOINT">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Joint</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="BTN_MOTION_PAUSE">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>10</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_GET_JOINT">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get
Cur Joint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_GET_TCP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get
Cur TCP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_GO_HOME">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>GoHome</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Setting</string>
   </property>
   <widget class="QPushButton" name="BTN_CHANGE_SPEED">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>change
speed</string>
    </property>
   </widget>
   <widget class="QSlider" name="SB_SPEED">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;speed [0~100%]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Real mode</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Gripper</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="BTN_GRIPPER_STOP">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_GRIPPER_OPEN">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BTN_GRIPPER_CLOSE">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="LE_GRIPPER_TIME">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>[0~600]</string>
    </property>
    <property name="text">
     <string>600</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>time [ms]</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BTN_BLEND_TEST">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>460</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>blend test</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
